### PR TITLE
Correct GraphQL variable name "text" to "type"

### PR DESCRIPTION
**Pull request by FumioNonaka-ksj**
In the CodeSandbox example of "Mutations in Apollo Client", the correct variable name is "type" instead of "text".
https://codesandbox.io/s/mutations-example-app-final-tjoje

### Checklist:

- [ ] If this PR is a new feature, please reference an issue where a consensus about the design was reached (not necessary for small changes)
- [ ] Make sure all of the significant new logic is covered by tests

### CODE REVIEW

**Comment by FumioNonaka-ksj** [comment]
The variable name is inconsistent with "[the completed version of the app](https://codesandbox.io/s/mutations-example-app-final-tjoje)". I spent more than an hour to have my trial app work.  Then, please revise "Mutations > Example app final" of CodeSandbox.

**Review by jpvajda — APPROVED** [review]
I went ahead and doubled checked this change and this PR is a good adjustment as the examples we provide in code sandbox do use `type` so to be consistent this is a good change, especially when someone is working through our code samples and docs to learn. We appreciate the effort to fix this! 

the complete version of the app is here: https://codesandbox.io/s/mutations-example-app-final-tjoje
the starter version of the app is here: https://codesandbox.io/s/mutations-example-app-start-gm7i5

they both use `type`

File: docs/source/data/mutations.mdx [hunk 1]
```diff
@@ -235,7 +235,7 @@ In most cases, a mutation response should include any object(s) the mutation mod
 {
   "__typename": "Todo",
   "id": "5",
-  "text": "Buy grapes 🍇"
+  "type": "Buy grapes 🍇"
```
Comment (benjamn) [inline docs/source/data/mutations.mdx:238]: This example seems a bit strange now. Maybe we change this to a more generic category?
```ts
"type": "groceries"
```
I can make this change unless anyone objects.

**Review by benjamn — APPROVED** [review]
Consistency across the tutorial materials is the goal, so I'm fine with switching to `type` instead of `text` as long as we're consistent!

File: docs/source/data/mutations.mdx (hunk 1 above)
Comment (jpvajda) [reply docs/source/data/mutations.mdx]: Good point. 🍇

**Comment by benjamn** [comment]
https://deploy-preview-9529--apollo-client-docs.netlify.app/data/mutations looks good to me!

**Pull request merged**